js/components/widgets/Title.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @function Title
+ * @typedef {Object} Title
+ * @property {string} priority
```

Comment:
Put the definition of your custom Title type in a separate JSDoc comment
Your Title custom type should have two properties: priority and e.g. text

Reply:
What do you mean by custom title?

Reply:
That's the type you came up with on your own

Reply:
* @typedef {Object} Title
* @property {number} priority
* @property {string} content